test/use-location.test.js

```diff
@@ -76,4 +76,15 @@ describe("`update` second parameter", () => {
     expect(history.length).toBe(histBefore + 1);
     unmount();
   });
+
+  it("replaces last entry with a new entry in the History object", () => {
+    const { result, unmount } = renderHook(() => useLocation());
+    const update = result.current[1];
+
+    const histBefore = history.length;
+    act(() => update("/about", true));
+
+    expect(history.length).toBe(histBefore);
```

Comment:
Can you also add another assertion here just to make sure the location has indeed been changed? 😄

Reply:
Of course, my bad 🙂

Reply:
Not your bad at all, you did everything right here. I just feel the assertion would make us more confident about the code.

Reply:
Done